3.2/test/run

```diff
@@ -341,6 +341,39 @@ run_doc_test() {
   echo
 }
 
+run_WT_cache_test() {
+    local name="WT_cache_size"
+    echo "  Testing setting of WT cache size"
+    local database='db'
+    local user='user'
+    local password='password'
+    local admin_password='adminPassword'
+
+    DOCKER_ARGS="
+-e MONGODB_DATABASE=${database}
+-e MONGODB_USER=${user}
+-e MONGODB_PASSWORD=${password}
+-e MONGODB_ADMIN_PASSWORD=${admin_password}
+"
+    # need to set these because `mongo_cmd` relies on global variables
+    USER=${user}
+    PASS=${password}
+    DB=${database}
+    ADMIN_PASS=${admin_password}
+
+    # minimum is 1G
+    create_container "${name}_200M" ${DOCKER_ARGS} -m 200M
+    test_connection ${name}_200M
+    mongo_admin_cmd "if (db.serverStatus()['wiredTiger']['cache']['maximum bytes configured'] == Math.pow(2,30)){quit(0)}; quit(1)"
+    # if greater that 1G, use 60% of (RAM - 1G)
+    create_container "${name}_5G" ${DOCKER_ARGS} -m 5G
+    test_connection ${name}_5G
+    mongo_admin_cmd "if (db.serverStatus()['wiredTiger']['cache']['maximum bytes configured'] == 2*Math.pow(2,30)){quit(0)}; quit(1)"
```

Comment:
this seems to be saying you expect the cache size to be 2 gigs, but the comment above implies it should be 60% of 4gigs(?), which is more than 2 gigs.  is it being rounded off?